nanshe_ipython.ipynb

```diff
@@ -1173,7 +1173,25 @@
         "                da_imgs_flt.dtype.itemsize >= 4):\n",
         "            da_imgs_flt = da_imgs_flt.astype(np.float32)\n",
         "\n",
-        "        da_result = normalize_data(da_imgs)\n",
+        "        da_imgs_flt_mins = da_imgs_flt.min(\n",
+        "            axis=tuple(irange(1, da_imgs_flt.ndim)),\n",
+        "            keepdims=True\n",
+        "        )\n",
+        "        da_imgs_flt_maxes = da_imgs_flt.max(\n",
+        "            axis=tuple(irange(1, da_imgs_flt.ndim)),\n",
+        "            keepdims=True\n",
+        "        )\n",
+        "\n",
+        "        da_imgs_flt = (\n",
+        "            (da_imgs_flt - da_imgs_flt_mins) /\n",
```

Comment:
The subtraction of the minimums is handy. That said, maybe it is better to subtract the minimum of the set instead.